zeroconf/__init__.py

```diff
@@ -1142,8 +1143,7 @@ def entries(self):
         else:
             # avoid size change during iteration by copying the cache
             values = list(self.cache.values())
-            return reduce(lambda a, b: a + b, values)
-
+            return list(itertools.chain.from_iterable(values))
```

Comment:
I didn't manage to figure out how to hit this in a profiled test. However Nallath himself seems to have profiler information on this change.